server/routes/summons-management/process-reply/process-reply.controller.js

```diff
@@ -5,6 +5,19 @@ const _ = require('lodash');
 const summonsValidator = require('../../../config/validation/summons-management');
 const validate = require('validate.js');
 
+module.exports.checkOwner = function(app) {
+  return function(req, res, next) {
+    if (req.session.jurorCommonDetails.owner !== req.session.authentication.locCode) {
```

Comment:
is jurorCommonDetails.owner available when we go to a summons? I think this only gets set if we visit the record no?

Reply:
Thanks, yes you're right. I've pushed a fix now.